profiles/default/S3-OIDC-google.cyberduckprofile

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2002-2017 iterate GmbH. All rights reserved.
+  ~ https://cyberduck.io/
+  ~
+  ~ This program is free software; you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation; either version 2 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  -->
+
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Protocol</key>
+        <string>s3</string>
+        <key>Vendor</key>
+        <string>s3-sts</string>
+        <key>Bundled</key>
+        <true/>
+        <key>Description</key>
+        <string>S3 STS AssumeRoleWithWebIdentity with Google</string>
+        <key>Default Nickname</key>
+        <string>S3 STS</string>
+        <key>Default Hostname</key>
+        <string>s3.amazonaws.com</string>
+        <key>Regions</key>
+        <array>
+            <string>eu-central-1</string>
```

Comment:
necessary?

Reply:
Can be omitted to allow all regions from AWS.